services/brig/brig.integration2.yaml

```diff
@@ -0,0 +1,164 @@
+brig:
+  host: 127.0.0.2
+  port: 8082
+
+cassandra:
+  endpoint:
+    host: 127.0.0.1
+    port: 9042
+  keyspace: brig_test
+
+elasticsearch:
+  url: http://127.0.0.1:9200
+  index: directory_test
+
+cargohold:
+  host: 127.0.0.2
```

Comment:
Because they really should only talk to the services in their own federation. They're just not up and running yet (need to add more lines to `intergration.sh` for that)